typescript/infra/config/environments/mainnet3/funding.ts

```diff
@@ -26,20 +26,21 @@ export const keyFunderConfig: KeyFunderConfig = {
   connectionType: RpcConsensusType.Single,
   // desired balance config
   desiredBalancePerChain: {
-    arbitrum: '3',
     avalanche: '5',
-    base: '3',
     bsc: '5',
     celo: '3',
-    ethereum: '5',
+    ethereum: '0.5',
```

Comment:
there's no boost for l1 iirc?

Reply:
yup, but lowering this back down due to levels of current use + cost of eth